### PR TITLE
Afterheader

### PR DESCRIPTION
**Pull request by airan1**
will add icons soon

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    </section>
+    <section class="middleinf">
+        <div class="container_middle">
+            <div class="main_content2">
+                <p class="context">WHY CHOOSE US</p>
+                <p class="context2">Problems trying to resolve the conflict between <br> 
+                the two major realms of Classical physics: Newtonian mechanics </p>
+            </div>
+            <div class="container">
+                <div class="cont1">
```
Comment (IvanCeban) [inline index.html:50]: WOW!
Didn't I say anything about numbers in class naming?
Please use more meaningful class names. 

I suggest you to rethink the grid-item namings.
Let's say what kind of data is that? If you take a closer look at the rest of the design you will notice that there are more of these cards.

So I suggest you to give each of the cards class card and use it for all sections in future.
The parent, the grid one contains cards. So you could give it a class called cards or cards-grid if you want to specify word "grid" there.
Inside each card you have "card header" and "card content". Here are the class names for them: card__header and card__content

Inside header you have icon and title.
So header__icn and header__title sound logical for their class names.

Basically you've chosen content agnostic class names as well. And you could use them for future elements. But "grid-item" doesn't tell us what kind of element is there. But "card" tells us that the element is a card and will look like a card. (it's a best practice to call cards elements like these)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+       <nav class="nav">
+        <div class="logo">
+            <h3>KMarketer</h3>
+        </div>
+            <ul class="nav_bar">
+                <li><a href="#">Home</a></li>
+                <li><a href="#">Product</a></li>
+                <li><a href="#">Pricing</a></li>
+                <li><a href="#">Contact</a></li>
+                <li><a><img alt="svg" src="img/Vector.png"></a></li>
```
Comment (IvanCeban) [inline index.html:27]: vector.png? 
C'mon... choose a better name

File: style.css [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+}
+
+.cont1,.cont2,.cont3,.cont4,.cont5,.cont6{
+    border: 1px solid;
+    background-color: rgb(255, 255, 255);
+    color: black;
+    text-align: center;
+}
+
+.context1p,.context2p,.context3p,.context4p,
```
Comment (IvanCeban) [inline style.css:193]: Why not adding just 1 common class to all these elements? 
What if you have 50 of same elements? You will give all of them personal class? context49p, context50p ?
I've told about common classes on last lesson

File: style.css [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+    background-color:#FFA62B;
+    border-color: #FFA62B;;
+    padding-left: 25px; /* Отступ слева */
+    padding-right: 25px;
+    margin-left: 20px;
+    transition: 0.2s;
+}
+
+.buttonquote2{
+    color: antiquewhite;
```
Comment (IvanCeban) [inline style.css:130]: the color is a very good candidate for a css variable

**Review by IvanCeban — CHANGES_REQUESTED** [review]
Please cleanup the class names. Better do it now until it became a complete mess

**Comment by airan1** [comment]
Снимок

File: index.html (hunk 1 above)
Comment (airan1) [reply index.html:50]: ok i will fix it

**Pull request merged**